src/Microsoft.AspNet.Razor/Generator/Compiler/CodeBuilder/CSharp/Visitors/CSharpCodeVisitor.cs

```diff
@@ -515,5 +515,28 @@ public static CSharpCodeWriter RenderPreWriteStart(CSharpCodeWriter writer, Code
 
             return writer;
         }
+
+        /// <summary>
+        /// Renders the Write method for TagHelpers.
+        /// </summary>
+        /// <param name="writer"><see cref="CSharpCodeWriter"/> to which the output must be written.</param>
+        /// <param name="context"><see cref="CodeBuilderContext"/> containing the necessary data to be written.</param>
+        /// <returns><see cref="CSharpCodeWriter"/> with the written content.</returns>
+        public static CSharpCodeWriter RenderWriteForTagHelperContent(
```

Comment:
nit @NTaylorMullen: could be used in `CSharpCodeVisitor.RenderRuntimeExpressionBlockChunk()` as well.  agree that it shouldn't live here unless we make that change.
either way, method isn't at all specific to tag helpers and should have a more general name.  perhaps `RenderStartWriteMethod()`.